mpsl/CHANGELOG.rst

```diff
@@ -9,6 +9,15 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nRF Connect SDK v1.5.1
+**********************
+
+Bug fixes
+=========
+
+* Fixed an issue where clock configuration option :c:member:`skip_wait_lfclk_started` in :c:struct:`mpsl_clock_lfclk_cfg_t`
+  might have not worked as expected with nRF5340 devices (DRGN-15223).
```

Comment:
Should use the same phrasing as in master. 
```suggestion
* Fixed an issue where the clock configuration option :c:member:`mpsl_clock_lfclk_cfg_t.skip_wait_lfclk_started` did not work as expected with nRF5340 devices (DRGN-15223).
```
You may cherry-pick https://github.com/nrfconnect/sdk-nrfxlib/pull/449/commits/363ebf03327d2850d9b98bbf4bd56fe5aa0d9430

Reply:
good catch